src/main/java/Backend.java

```diff
@@ -293,7 +251,7 @@ public static void main(String[] args) {
         try {
             boolean send_binary_test_message = Boolean.parseBoolean(args[2]);
             if (send_binary_test_message) {
-                Backend backend = new Backend("JavaNode", "javamailbox");
+                Backend backend = new Backend("JavaNode", "javamailbox", "antidote");
```

Comment:
Personally I'm a big fan of putting Main in its own file, because it opens up the opportunity to do tricks (like the TestingBackend suggested above), but also because it is a natural starting point to look at the code.